dbaas-lib/k8s/cmd.go

```diff
@@ -52,6 +52,19 @@ type ErrCmdRun struct {
 	output []byte
 }
 
+type Service struct {
+	Status ServiceStatus `json:"status"`
+}
+
+type ServiceStatus struct {
+	LoadBalancer struct {
+		Ingress []struct {
+			IP       string `json:"ip"`
+			Hostname string `json:"hostname"`
+		} `json:"ingress"`
+	} `json:"loadBalancer"`
+}
+
```

Comment:
I don't see any usage of this types

Reply:
https://github.com/Percona-Lab/percona-dbaas-cli/blob/DBAASCLI-68/dbaas-lib/engines/k8s-pxc/controller.go#L119

Reply:
Why not use https://pkg.go.dev/k8s.io/api/core/v1?tab=doc#ServiceStatus ?

Reply:
done in https://github.com/Percona-Lab/percona-dbaas-cli/pull/102/commits/7dfd4afa3d703237ee815bc948e147cf1fb48dad